lib/tmuxinator/config.rb

```diff
@@ -14,12 +14,11 @@ def sample
         asset_path "sample.yml"
       end
 
-      def default
-        "#{ENV['HOME']}/.tmuxinator/default.yml"
-      end
-
-      def default?
-        exists?("default")
+      def default(filename)
+        if exists? (filename)
```

Comment:
The one-liner syntax that RuboCop adores would have this method look like the following, since it generally sides against having methods be only an if/else statement:

```ruby
      def default(filename) 
        return "#{ENV['HOME']}/.tmuxinator/#{filename}.yml" if exists?(filename)
        sample
      end
```

At least by default, that is. We may have customized RuboCop.